### PR TITLE
[v3 Branch Pick] Exclude all 4xx Rails errors

### PR DESCRIPTION
**Pull request by st0012**
This PR picks the changes of https://github.com/getsentry/raven-ruby/pull/769

### CODE REVIEW

File: lib/raven/configuration.rb [hunk 1]
```diff
@@ -172,16 +172,29 @@ class Configuration
...
+      'ActionController::InvalidCrossOriginRequest',
+      'ActionController::MethodNotAllowed',
+      'ActionController::NotImplemented',
+      'ActionController::ParameterMissing',
       'ActionController::RoutingError',
       'ActionController::UnknownAction',
+      'ActionController::UnknownFormat',
+      'ActionController::UnknownHttpMethod',
+      'ActionDispatch::Http::Parameters::ParseError',
+      'ActionView::MissingTemplate',
```
Comment (agrobbin) [inline lib/raven/configuration.rb:190]: @st0012 I'm not sure what the current state of the `v3` branch is, but the inclusion of `ActionView::MissingTemplate` here feels like a bug. While Rails does render a `406 Not Acceptable` when [`ActionView::MissingExactTemplate`](https://github.com/rails/rails/blob/v6.0.3.4/actionpack/lib/action_dispatch/middleware/exception_wrapper.rb#L16) is raised, it renders a `500 Internal Server Error` when `ActionView::MissingTemplate` is raised (via its [default error handling](https://github.com/rails/rails/blob/v6.0.3.4/actionpack/lib/action_dispatch/middleware/exception_wrapper.rb#L8)).

Comment (st0012) [reply lib/raven/configuration.rb:190]: yeah I think it's too aggressive to exclude the `ActionView::MissingTemplate` by default. will remove it in the upcoming version. thanks for the opinion 👍

Comment (st0012) [reply lib/raven/configuration.rb:190]: I've added https://github.com/getsentry/sentry-ruby/pull/1092 to remove it

Comment (agrobbin) [reply lib/raven/configuration.rb:190]: Thanks @st0012!!

**Pull request merged**